ivadomed/scripts/extract_small_dataset.py

```diff
@@ -99,47 +102,47 @@ def extract_small_dataset(input, output, n=10, contrast_list=None, include_deriv
     for subject in subject_random_list:
         print("\nSubject: {}".format(subject))
         # Copy images
-        isubjfolder = os.path.join(input, subject)
-        osubjfolder = os.path.join(output, subject)
-        assert os.path.isdir(isubjfolder)
-        print("\tCopying {} to {}.".format(isubjfolder, osubjfolder))
-        shutil.copytree(isubjfolder, osubjfolder)
+        in_subjfolder = Path(input, subject)
+        out_subjfolder = Path(output, subject)
+        assert in_subjfolder.is_dir()
```

Comment:
Could use use "_" to separate "subj" and "folder" as well here and below as well?